### PR TITLE
os: build bcast address interface.

### PR DESCRIPTION
**Pull request by Txiaozhe**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added

### CODE REVIEW

**Comment by Txiaozhe** [comment]
I am sorry but I redo it! @yorkie

**Comment by Txiaozhe** [comment]
Thanks!

File: test/run_pass/test_os.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+var os = require('os');
+var interfaces = os.networkInterfaces();
+var actual = {};
+
+switch (os.platform()) {
+  case 'linux': {
+    actual = interfaces.lo.filter(function (e) {
+      return e.address === '127.0.0.1' &&
+              e.netmask === '255.0.0.0' &&
+               e.family === 'IPv4';
```
Comment (yorkie) [inline test/run_pass/test_os.js]: Just a small nits, keep alignments :)

File: test/run_pass/test_os.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    });
+    break;
+  }
+  case 'darwin': {
+    actual = interfaces.lo0.filter(function (e) {
+      return e.address === '127.0.0.1' &&
+              e.netmask === '255.0.0.0' &&
+              e.family === 'IPv4';
```
Comment (yorkie) [inline test/run_pass/test_os.js]: Ditto..

**Comment by Txiaozhe** [comment]
Thanks！ @yorkie

**Pull request merged**